src/Randomizer.SMZ3.Tracking/AutoTracking/MemoryDocumentation.md

```diff
@@ -0,0 +1,40 @@
+﻿# Memory Documentation
```

Comment:
Thank you for this! If you rename it to README you'll see it on GitHub when in this folder, too.